### DISCUSSION TITLE
推荐个文献管理软件

### DISCUSSION THREAD

**Discussion by zhoujian9410** [nju-lug/NJUThesis#81] [Show and tell]
文档里面推荐了Endnote和Zotero，这两个软件之前我都用过，但是个人感觉和Latex配合不太好。
最后用的是老师推荐的“JabRef”。它直接管理bib文件，不需要像别的软件一样导出bib，还可以向Texstudio之类外部软件推送引用命令加bibtexkey。
而且还是免费软件，Github就可以下载，[JabRef](https://github.com/JabRef/jabref/releases)

个人感觉还算好用，供大家参考

**Comment by atxy-blip** [comment]
感谢推荐，稍后会在手册中增加相关介绍。

-----

另：Zotero 可以借助 [Better BibTeX 插件](https://github.com/retorquere/zotero-better-bibtex)实现更多支持功能。
